src/FSharp.Control.TaskSeq/Utils.fsi

```diff
@@ -66,8 +69,8 @@ module Task =
     val inline toValueTask: task: Task<'T> -> ValueTask<'T>
 
     /// <summary>
-    /// Convert a ValueTask&lt;'T> to a Task&lt;'T>. To use a non-generic ValueTask,
```

Comment:
I was unable to parse this on multiple reads the way it was. Not 100% sure it's better, but I hope so.

Reply:
Yeah, all these "generic vs non-generic" versions of things in .NET are hard to explain simply. I've had several version of this text in the past. Each improvement is welcome! 👍